runtimes/pythonrt/py-sdk/autokitteh/oauth2_session.py

```diff
@@ -0,0 +1,58 @@
+import requests
+from os import getenv
+
+from .connections import refresh_oauth
+from .connections import check_connection_name
```

Comment:
FYI, this won't pass Ruff's formatting, but we're not enforcing that in the autokitteh repo yet, so no need to fix it now.

Reply:
this is a result of ruff...